src/app/fdctl/config/default.toml

```diff
@@ -158,9 +158,23 @@ dynamic_port_range = "8000-10000"
     # `--rpc-pubsub-enable-vote-subscription` argument.
     pubsub_enable_vote_subscription = false
 
+# The Solana Labs client periodically takes snapshots of the chain's
+# state that other clients can query to catch up to the current state.
```

Comment:
```suggestion
# state so that other clients can query to catch up to the current state.
```

Reply:
I reworded this sentence. Let me know what you think.